forms-flow-web/src/components/Form/Item/View.js

```diff
@@ -170,7 +174,24 @@ const View = React.memo((props) => {
   const saveDraft = (payload) => {
     let dataChanged = !isEqual(payload.data, lastUpdatedDraft.data);
     if (draftSubmissionId && isDraftCreated) {
-      if (dataChanged) dispatch(draftUpdateMethod(payload, draftSubmissionId));
+      if (dataChanged) {
+        setDraftCreating(true);
+        dispatch(
+          draftUpdateMethod(payload, draftSubmissionId, (err) => {
+            if (err) {
+              toast.error(err);
```

Comment:
This is a bad indicator.

Reply:
Remove Toast for Draft . Will find some better alternative say offline  Case